test/loading.jl

```diff
@@ -88,13 +88,13 @@ using Test
 
 shastr = "ab"^20
 hash = SHA1(shastr)
-@test hash == eval(sprint(io -> show(io, hash))) # check show method
+@test hash == eval(parse(sprint(io -> show(io, hash)))) # check show method
```

Comment:
Isn't `string(hash)` equivalent to `sprint(show, hash)` which is equivalent to `sprint(io -> show(io, hash))`?

Reply:
The purpose of this PR was to make `string(hash)` and `show(hash)` different. I didn't think of using `sprint(show, hash)` though, good call.